ui/src/lib.ts

```diff
@@ -4,6 +4,7 @@ import { EventInputTransformer, EventInput } from "@fullcalendar/react";
 const padNumberWith0Zero: Function = (num: Number): string =>
   num.toString().padStart(2, "0");
 
+// ex. 2022-01-15
```

Comment:
same here does ex mean example and it s this documentation that can be made clear through viewing the tests for these functions?

Reply:
Updated here to include a description 8e55de2